src/databricks/labs/remorph/config.py

```diff
@@ -1,8 +1,12 @@
 import logging
 from dataclasses import dataclass
 
+from pyspark.sql import DataFrame
 from sqlglot.dialects.dialect import Dialect, Dialects
 
+from databricks.labs.remorph.reconcile.query_builder_refactored.recon_config import (
+    Table,
```

Comment:
fix this import format

Reply:
I tried fixing this ,but our formatter re-formats to this version